src/pykeen/nn/modules.py

```diff
@@ -1018,7 +1018,7 @@ class TorusEInteraction(TranslationalInteraction[torch.FloatTensor, torch.FloatT
 
     func = pkf.toruse_interaction
 
-    def __init__(self, p: int = 2, power_norm: bool = True):
+    def __init__(self, p: int = 2, power_norm: bool = False):
```

Comment:
I think this matches the base class's constructor so you might be able to omit it completely

Reply:
The default values are new 🙂 

But I would also support moving them to the base class.

Reply:
Hmm they don't all specify this. So let's leave it as it is for now